### ISSUE TITLE
CI: Analyze #includes in source files fails: sudo: dnf: command not found

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylladb#19774]
Seen in https://github.com/scylladb/scylladb/actions/runs/9888238804/job/27311706240?pr=16827
```
> Run sudo dnf -y install clang-tools-extra
sudo dnf -y install clang-tools-extra
  shell: /usr/bin/bash -e {0}
  env:
    BUILD_TYPE: RelWithDebInfo
    BUILD_DIR: build
    CLEANER_OUTPUT_PATH: build/clang-include-cleaner.log
    CLEANER_DIRS: test/unit exceptions alternator api auth cdc compaction
sudo: dnf: command not found
Error: Process completed with exit code 1.
```

This was introduced in https://github.com/scylladb/scylladb/pull/19628 (229250ef3e5007e4adedbbebcfe3723c64cbd5a1)

**Comment by tchaikov** [comment]
@bhalevy this has been fixed by 2a1c9ed7cb36fceb2fa29d9457fce5928b15c443 . could you please rebase your change against a commit which include this commit. i see the HEAD of your pull request share the ancestor of 9a71543fd29e3ac181b41b964ea870bd5fb742e6 with master. but 9a71543fd29e3ac181b41b964ea870bd5fb742e6 does not include 2a1c9ed7cb36fceb2fa29d9457fce5928b15c443 .

**Comment by bhalevy** [comment]
> @bhalevy this has been fixed by [2a1c9ed](https://github.com/scylladb/scylladb/commit/2a1c9ed7cb36fceb2fa29d9457fce5928b15c443) . could you please rebase your change against a commit which includes this commit. i see the HEAD of your pull request shares the ancestor of [9a71543](https://github.com/scylladb/scylladb/commit/9a71543fd29e3ac181b41b964ea870bd5fb742e6) with master. but [9a71543](https://github.com/scylladb/scylladb/commit/9a71543fd29e3ac181b41b964ea870bd5fb742e6) does not include [2a1c9ed](https://github.com/scylladb/scylladb/commit/2a1c9ed7cb36fceb2fa29d9457fce5928b15c443) .

It works after rebase.  So this issue can be closed.
